purchase_orderline_images/views/purchase_order_view.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record
+            id="purchase_order_form_product_image_purchase_report"
```

Comment:
Inherited view must use the same record is as its parent view
Check [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml)
So id must be "purchase.purchase_order_form"